Adding 2004 Klamath primary and general results

JasonBernert opened the pull request.

nk9 commented:
@JasonBernert, would you mind running the verifier script on these new files? It finds a few issues at the moment. Should be easy to fix.

JasonBernert commented:
@nk9 Good catch! I just ran `verifier.py` on all Klamath results I've added and fixed any errors. I'm doing the 2002 Klamath general results today. I'll include everything in the next pull request.
